cmd/stack_select.go

```diff
@@ -27,6 +27,8 @@ import (
 // newStackSelectCmd handles both the "local" and "cloud" scenarios in its implementation.
 func newStackSelectCmd() *cobra.Command {
 	var stack string
+	var secretsProvider string
```

Comment:
We may want to provide a `secrets-provider` flag as well (see https://github.com/pulumi/pulumi/blob/master/cmd/stack_init.go#L116-L118).